### ISSUE TITLE
Class blade.compiler does not exist

### ISSUE THREAD

**Issue by dorianboulch** [albertcht/invisible-recaptcha#28]
Hi
I'm running Laravel v5.2 and when I execute ``composer update`` I have this stacktrace :
```
[2017-07-02 15:41:56] local.ERROR: exception 'ReflectionException' with message 'Class blade.compiler does not exist' in /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php:734
Stack trace:
#0 /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php(734): ReflectionClass->__construct('blade.compiler')
#1 /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php(629): Illuminate\Container\Container->build('blade.compiler', Array)
#2 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(697): Illuminate\Container\Container->make('blade.compiler', Array)
#3 /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php(573): Illuminate\Foundation\Application->make('Illuminate\\View...')
#4 /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php(533): Illuminate\Container\Container->addDependencyForCallParameter(Object(ReflectionParameter), Array, Array)
#5 /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php(505): Illuminate\Container\Container->getMethodDependencies(Array, Array)
#6 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(757): Illuminate\Container\Container->call(Array)
#7 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(740): Illuminate\Foundation\Application->bootProvider(Object(AlbertCht\InvisibleReCaptcha\InvisibleReCaptchaServiceProvider))
#8 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(AlbertCht\InvisibleReCaptcha\InvisibleReCaptchaServiceProvider), 16)
#9 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(741): array_walk(Array, Object(Closure))
#10 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\Foundation\Application->boot()
#11 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#12 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(219): Illuminate\Foundation\Application->bootstrapWith(Array)
#13 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(105): Illuminate\Foundation\Console\Kernel->bootstrap()
#14 /var/www/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 {main}
```

If I remove the ``InvisibleReCaptchaServiceProvider`` from my providers, I can update successfuly.

**Comment by albertcht** [comment]
hi @dorianboulch 
This bug is fixed in the latest release v1.6. Can you update your package and try it again?

**Comment by dorianboulch** [comment]
Yes !
It's all good. Thanks

Maybe you should update composer default version ?
